app/vmalert/notifier/alertmanager_test.go

```diff
@@ -112,3 +113,95 @@ func TestAlertManager_Send(t *testing.T) {
 		t.Errorf("expected 2 calls(count from zero) to server got %d", c)
 	}
 }
+
+func TestAlertManager_SendWithAuthorization(t *testing.T) {
```

Comment:
The same problem here as in previous test. Either extend original test function or come up with a simpler test which check auth params only.

Reply:
removed, implemented extended test